launch.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "0.2.0",
```

Comment:
Add both `extensions.json` and `launch.json` to `.gitignore`. We don't usually check in these files.